.circleci/config.yml

```diff
@@ -239,17 +318,21 @@ workflows:
                  - dev
     jobs:
       - build
-      - integration-presubmit-kind
+      - integration-presubmit-one-namespace-kind
+      - integration-presubmit-isolated-namespaces-kind
       - integration-old-kind
       - install-minikube
       - install-demo
-      - noauth
+      - noauth_one_namespace
+      - noauth_isolated_namespaces
```

Comment:
Also: if you want to test this case, no need to create a new job - each job can test different scenarios in different namespaces. For local testing - we do need to keep distinct cluster for 'noauth' type of tests because we'll not 
have citadel and need to test that everything works in the absence of certs. But multiple install profiles that 
are in this situation can use the same cluster - we also want to verify they don't interfere with each other. 

( we are still short on resources - each job takes ~10.15 min I think - if you really want this profile as a separate job, maybe add it to nightly ?)